org.omg.kerml.expressions.xtext.ui/plugin.xml_gen

```diff
@@ -330,68 +330,6 @@
 			</command>
 		</menuContribution>
 	</extension>
-	<!-- quickfix marker resolution generator for org.omg.kerml.expressions.xtext.KerMLExpressions -->
```

Comment:
Please check whether these changes need to be copied to the plugin.xml file of the same project. The basic idea here is that Xtext is not capable of updating the plugin.xml files (as it could contain manual customizations as well), so an alternative version Is generated where the developers are expected to copy the changes.